From 4dce330c043db6671d558e486ce5be17facae574 Mon Sep 17 00:00:00 2001
From: Evelyn Torres <lostentrepreneur439@yahoo.com>
Date: Tue, 19 May 2026 10:40:47 -0230
Subject: [PATCH] config: Nuke Go support

Go support is broken in crDroid, and causes issues on devices like hammerhead
---
 config/common_mobile.mk | 8 --------
 1 file changed, 8 deletions(-)

diff --git a/config/common_mobile.mk b/config/common_mobile.mk
index 5c15ad1..9c843f7 100644
--- a/config/common_mobile.mk
+++ b/config/common_mobile.mk
@@ -20,19 +20,11 @@ PRODUCT_PACKAGES += \
     Glimpse \
     LatinIME
 
-ifeq ($(PRODUCT_TYPE), go)
-PRODUCT_PACKAGES += \
-    Launcher3QuickStepGo
-
-PRODUCT_DEXPREOPT_SPEED_APPS += \
-    Launcher3QuickStepGo
-else
 PRODUCT_PACKAGES += \
     Launcher3QuickStep
 
 PRODUCT_DEXPREOPT_SPEED_APPS += \
     Launcher3QuickStep
-endif
 
 PRODUCT_PACKAGES += \
     Launcher3Overlay
-- 
2.53.0

